pvm.py

```diff
@@ -43,9 +43,13 @@ def parse_commands(*args):
 		pass
 	# TODO: Create another python file to control two display
 	if command=="file":
-		_logger.info("File set: %s", PEFIX_PATH + value)
+		_logger.info("File set: %s", PREFIX_PATH + value)
```

Comment:
I think this log should be moved to line 51.

Reply:
agreed. @SSSSam141 please make that simple move. No need to print that file was set if it is in fact not being set twice.

Reply:
Done